### ISSUE TITLE
Fields are not submitted in e-mail after validation

### ISSUE THREAD

**Issue by mpr92** [formvalidation/support#862]
Hello everybody,
I have an acute problem. Namely, I integrated the contact form and the FormValidation about a year ago. Recently, we noticed that the e-mails are sent via the contact form, but the input fields are not transferred and all are empty.

I have already reviewed countless possibilities, but I can not find the mistake. Is the field transfer with the POST correct or does it have to be transferred differently with jQuery?
Maybe you can help me or have an idea why it is:

`<section class="bg-brown-light" id="bestellformular" style="padding-bottom: 0">
        <article class="container">
          <div class="row section-100">
            <div class="col-md-6 col-md-offset-3">
							
	     <h2>Jetzt direkt bestellen und anfragen!</h2>
              <p class="subtitle"></p>
                            
              
              <div id="alertContainer"></div>
  
              <form id="orderForm" method="post" class="form-horizontal" accept-charset="UTF-8">
              
                  <div class="form-group">
                      <div class="radio col-xs-12 col-sm-4" style="padding-left: 35px;">
                          <input type="radio" name="frmTyp" value="Anfrage" id="frmTypRequest" class="styled" />
                          <label for="frmTypRequest" class="h3">Anfrage</label>
                      </div>
                
                      <div class="radio col-xs-12 col-sm-8" style="padding-left: 35px;">
                          <input type="radio" name="frmTyp" value="Bestellung" id="frmTypOrder" class="styled"  />
                          <label for="frmTypOrder" class="h3">Bestellung</label>
                      </div>
                  </div>
                  
                  <div class="form-group">
                      <label class="col-xs-12 col-sm-4 control-label">Vor- / Nachname</label>
                      <div class="col-xs-8 col-sm-4">
                          <input type="text" class="form-control" name="firstName" placeholder="" />
                          <span class="help-block" id="firstNameMessage"></span>
                      </div>
                      <div class="col-xs-4 col-sm-4">
                          <input type="text" class="form-control" name="lastName" placeholder="" />
                          <span class="help-block lastNameMessage"></span>
                      </div>
                  </div>
                  
                  <div class="form-group">
                      <label class="col-xs-12 col-sm-4 control-label">Straße / Nr.</label>
                      <div class="col-xs-8 col-sm-6">
                          <input type="text" class="form-control" name="street" placeholder="" />
                          <span class="help-block" id="streetMessage"></span>
                      </div>
                      <div class="col-xs-4 col-sm-2">
                          <input type="text" class="form-control" name="streetNumber" placeholder="" />
                          <span class="help-block streetNumberMessage"></span>
                      </div>
                  </div>
                  
                  <div class="form-group">
                      <label class="col-xs-12 col-sm-4 control-label">PLZ / Ort</label>
                      <div class="col-xs-4 col-sm-4">
                          <input type="text" class="form-control" name="plz" placeholder=""  />
                          <span class="help-block" id="plzMessage"></span>
                      </div>
                      <div class="col-xs-8 col-sm-4">
                          <input type="text" class="form-control" name="city" placeholder="" />
                          <span class="help-block cityMessage"></span>
                      </div>
                  </div>
              
                  <div class="form-group">
                      <label class="col-xs-12 col-sm-4 control-label">Telefonnummer</label>
                      <div class="col-xs-12 col-sm-4">
                          <input type="text" class="form-control" name="phoneNumber" />
                      </div>
                  </div>
              
                  <div class="form-group">
                      <label class="col-xs-12 col-sm-4 control-label">E-Mail-Adresse</label>
                      <div class="col-xs-12 col-sm-4">
                          <input type="text" class="form-control" name="email" />
                      </div>
                  </div>
                  
                  <fieldset>
                    
                    <legend class="control-label">Liköre</legend>
                    <p class="text">Flaschengrößen: 50ml, 200ml, 350ml, 500ml <button type="button" class="btn btn-info product-indication" data-toggle="modal" data-target="#vorschauGroessen"><i class="fa fa-question" aria-hidden="true"></i></button></p>
                    <div class="row">
                      <div class="col-sm-12">
                        <div class="form-group">
                            <label class="col-xs-6 control-label">Allewetter Stimmungsretter</label>
                            <div class="col-xs-6">
                                <input type="text" class="form-control" name="allewetterstimmungsretter" placeholder="Bsp: 2x 350ml, 10x50ml"  />
                            </div>
                        </div>
                         ...
                      </div>
                    </div>
                  </fieldset>
                  
                  <div class="form-group">
                      <div class="col-xs-12">
                          <textarea class="form-control" name="message" rows="2" placeholder="Haben Sie noch Anmerkungen?"></textarea>
                      </div>
                  </div>
                  ...
              </form>
              
              
              </div>
            </div>
        </artcile>
      </section>
   
        
        <div class="modal fade" id="vorschauGroessen" tabindex="-1" role="dialog" aria-labelledby="vorschauGroessenLabel">
          <div class="modal-dialog" role="document">
            <div class="modal-content">
              <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal" aria-label="Schließen"><span aria-hidden="true">&times;</span></button>
                <h4 class="modal-title" id="vorschauGroessenLabel">Vorschau der Flaschengrößen</h4>
              </div>
              <div class="modal-body">
                <img src="assets/beispielgroessen_likoere_mengen.jpg" class="img-responsive" alt="Vorschau Likör Flaschengrößen" title="Vorschau der Flaschengrößen">
              </div>
              
            </div>
          </div>
        </div>
      
    </main>

		 
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script> 

		
		<script src="/js/bootstrap.min.js"></script>
		<script src="/js/functions.js"></script>

		
		<script src="/js/jquery.easing.min.js"></script>
		<script src="/js/wow.js"></script>
		<script src="/js/modernizr.js"></script>
		<script src="/js/scripts.js"></script>
		
		
		<script src="/plugins/formvalidation/formValidation.min.js"></script>
		<script src="/plugins/formvalidation/framework/bootstrap.min.js"></script>

		
		<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.form/3.51/jquery.form.min.js"></script>

		
		<script src="/plugins/formvalidation/language/de_DE.js"></script>
    <script>
      $(document).ready(function() {
          // Generate a simple captcha
          function randomNumber(min, max) {
              return Math.floor(Math.random() * (max - min + 1) + min);
          }
      
          function generateCaptcha() {
              $('#captchaOperation').html([randomNumber(1, 100), '+', randomNumber(1, 200), '='].join(' '));
          }
      
          generateCaptcha();
      
          $('#orderForm').formValidation({
                framework: 'bootstrap',
                icon: {
                    valid: 'fa fa-check',
                    invalid: 'fa fa-times',
                    validating: 'fa fa-refresh'
                },
                excluded: [':disabled'],
                fields: {
                    frmTyp: {
                        validators: {
                            notEmpty: {
                                message: 'Bitte Typ wählen'
                            }
                        }
                    },
                    firstName: {
                        err: '#firstNameMessage',
                        validators: {
                            notEmpty: {
                                message: 'Bitte Vorname angeben'
                            }
                        }
                    },
                    lastName: {
                        err: '.lastNameMessage',
                        validators: {
                            notEmpty: {
                                message: 'Bitte Nachname angeben'
                            }
                        }
                    },
                    street: {
                        err: '#streetMessage',
                        validators: {
                            notEmpty: {
                                message: 'Bitte Straße angeben'
                            },
                        }
                    },
                    streetNumber: {
                        err: '.streetNumberMessage',
                        validators: {
                            integer: {
                                message: 'Bitte Hausnummer angeben'
                            }
                        }
                    },
                    plz: {
                        err: '#plzMessage',
                        validators: {
                            integer: {
                                message: 'Keine gültige PLZ'
                            },
                            stringLength: {
                                min: 5,
                                max: 5
                            }
                        }
                    },
                    city: {
                        err: '.cityMessage',
                        validators: {
                            notEmpty: {
                                message: 'Bitte Ort angeben'
                            }
                        }
                    },
                    phoneNumber: {
                        enabled: false,
                        validators: {
                            notEmpty: {
                                message: 'Bitte Telefonnummer angeben'
                            },
                            regexp: {
                                message: 'Die Telefonnummer darf nur Ziffern, Leerzeichen, -, + oder . enthalten',
                                regexp: /^[0-9\s\-()+\.]+$/
                            }
                        }
                    },
                    email: {
                        validators: {
                            notEmpty: {
                                message: 'Bitte E-Mai-Adresse angeben'
                            },
                            emailAddress: {
                                message: 'Bitte gültige E-Mail-Adresse angeben'
                            }
                        }
                    },
                  
                  
                    allewetterstimmungsretter: {
                        enabled: false,
                        validators: {
                            notEmpty: {
                                message: ''
                            }
                        }
                    },
                   ....            
                    
                    title: {
			enabled: false,
			validators: {
                            notEmpty: {
                                message: 'Bitte einen Namen wählen'
                            }
                        }
                    },
                    otherTitle: {
                      enabled: false,
			validators: {
                            callback: {
                                message: 'Please specific the label',
                                callback: function(value, validator, $field) {
                                    var title = $('#orderForm').find('[name="title"]:checked').val();
                                    return (title !== 'Vornamen') ? true : (value !== '');
                                }
                            }
                        }
                    },
                    slogan: {
                      enabled: false,
			validators: {
                            notEmpty: {
                                message: 'Please select a label'
                            }
                        }
                    },
                    otherSlogan: {
                      enabled: false,
			validators: {
                            callback: {
                                message: 'Please specific the label',
                                callback: function(value, validator, $field) {
                                    var slogan = $('#orderForm').find('[name="slogan"]:checked').val();
                                    return (slogan !== 'Trauspruch') ? true : (value !== '');
                                }
                            }
                        }
                    },
                    date: {
                      enabled: false,
			validators: {
                            notEmpty: {
                                message: 'Wir benötigen Ihr Traudatum'
                            }
                        }
                    },
                    dateWedding: {
                      enabled: false,
			validators: {
                            callback: {
                                message: 'Wir benötigen Ihr Traudatum',
                                callback: function(value, validator, $field) {
                                    var date = $('#orderForm').find('[name="date"]:checked').val();
                                    return (date !== 'Datum') ? true : (value !== '');
                                }
                            }
                        }
                    },
                    delivery: {
                        validators: {
                            notEmpty: {
                                message: 'Angabe gewünschter Liefertermin'
                            }
                        }
                    },
                    dateDelivery: {
                        validators: {
                            callback: {
                                message: 'Liefertermin fehlt',
                                callback: function(value, validator, $field) {
                                    var delivery = $('#orderForm').find('[name="delivery"]:checked').val();
                                    return (delivery !== 'Liefertermin') ? true : (value !== '');
                                }
                            }
                        }
                    },									
                    message: {
                        enabled: false,
                        validators: {
                            notEmpty: {
                                message: 'Was möchten Sie uns mitteilen?'
                            },
                            stringLength: {
                                max: 700,
                                message: 'Bitte verwenden Sie nicht mehr als 700 Zeichen'
                            }
                        }
                    },
                    'age[]': {
                        validators: {
                            notEmpty: {
                                message: 'Bitte das Mindestalter von 18 Jahren bestätigen'
                            }
                        }
                    },
                    captcha: {
                        validators: {
                            callback: {
                                message: 'Falsche Antwort',
                                callback: function(value, validator, $field) {
                                    var items = $('#captchaOperation').html().split(' '),
                                        sum   = parseInt(items[0]) + parseInt(items[2]);
                                    return value == sum;
                                }
                            }
                        }
                    }
                }
            })
            .on('change', '[name="title"]', function(e) {
                $('#orderForm').formValidation('revalidateField', 'otherTitle');
            })
            .on('change', '[name="slogan"]', function(e) {
                $('#orderForm').formValidation('revalidateField', 'otherSlogan');
            })
            .on('change', '[name="date"]', function(e) {
                $('#orderForm').formValidation('revalidateField', 'dateWedding');
            })
            .on('change', '[name="label"]', function(e) {
                $('#orderForm').formValidation('revalidateField', 'otherChannel');
            })
            .on('success.field.fv', function(e, data) {
                if (data.field === 'otherTitle') {
                    var title = $('#orderForm').find('[name="title"]:checked').val();
                    // User choose given label
                    if (title !== 'Vornamen') {
                        // Remove the success class from the container
                        data.element.closest('.form-group').removeClass('has-success');
    
                        // Hide the tick icon
                        data.element.data('fv.icon').hide();
                    }
                };
                if (data.field === 'otherSlogan') {
                    var slogan = $('#orderForm').find('[name="slogan"]:checked').val();
                    // User choose given label
                    if (slogan !== 'Trauspruch') {
                        // Remove the success class from the container
                        data.element.closest('.form-group').removeClass('has-success');
    
                        // Hide the tick icon
                        data.element.data('fv.icon').hide();
                    }
                };
                if (data.field === 'dateWedding') {
                    var date = $('#orderForm').find('[name="date"]:checked').val();
                    // User choose given label
                    if (date !== 'Datum') {
                        // Remove the success class from the container
                        data.element.closest('.form-group').removeClass('has-success');
                        // Hide the tick icon
                        data.element.data('fv.icon').hide();
                    }
                };
                if (data.field === 'dateDelivery') {
                    var delivery = $('#orderForm').find('[name="delivery"]:checked').val();
                    // User choose given label
                    if (delivery !== 'Liefertermin') {
                        // Remove the success class from the container
                        data.element.closest('.form-group').removeClass('has-success');
                        // Hide the tick icon
                        data.element.data('fv.icon').hide();
                    }
                }
            })
            .on('err.field.fv', function(e, data) {			
...
            })
            .on('err.form.fv', function(e) {
                // Regenerate the captcha
                generateCaptcha();
            })
            .on('success.form.fv', function(e) {
                // Prevent default form submission
                e.preventDefault();
          
                var $form = $(e.target);
                
                // Send all form data to back-end
                $form.ajaxSubmit({
                    url: 'send.php',
                    dataType: 'json',
                    success: function(response) {
                        // Clear the form
                        $form.formValidation('resetForm', true);
          
                        // Regenerate the captcha
                        generateCaptcha();
          
                         // Show the message
                        response.result === 'error'
                            ? $('#alertContainer')      
                                .removeClass('alert-success')
                                .addClass('alert-warning')
                                .html('Sorry, cannot send the message')
                                .show()
                            : $('#alertContainer')
                                .removeClass('alert-warning')
                                .addClass('alert-success')
                                .html('Your message has been successfully sent')
                                .show();
                    }
                });
            });
        });
      </script>
`
And my "send.php" looks like this:
<?php
	//$empfaenger = "info@goldegg-icm.de";
	$empfaenger = "m-pruss@email.de";
	//$ccadmin = "CC: kontakt@mpruss.de";
	
	$antwortan = $_POST['email'];
	$absendername = "Goldegg. Genuss. Lifestyle.";
	$absendermail = $_POST['email'];
	$betreff = "".$_POST['frmTyp']." von goldegg-genuss.de";
	$text = "Es ist eine neue ".$_POST['frmTyp']." eingegangen:\n
	
	Name: ".$_POST['firstName']." ".$_POST['lastName']."\n 
	Straße: ".$_POST['street']." ".$_POST['streetNumber']."\n 
	PLZ / Ort: ".$_POST['plz']." ".$_POST['city']."\n 
	Telefon: ".$_POST['phoneNumber']."\n
	E-Mail: ".$_POST['email']."\n\n
	
	Folgende Bestellung ist eingegangen:\n
	Allewetter Stimmungsretter: ".$_POST['allewetterstimmungsretter']."
	Ehe-Glück: ".$_POST['eheglueck']."
	Ehe-Glück Kräuter: ".$_POST['eheglueckkraeuter']."
	IBBKiss: ".$_POST['ibbkiss']."
	Ibbenbürener Kohlengeist: ".$_POST['ibbenbuerenerkohlengeist']."
	Laggenbecker Lecker: ".$_POST['laggenbeckerlecker']."
	OONALOOBA: ".$_POST['oonalooba']."
	Recker Moorgeister: ".$_POST['reckermoorgeister']."
	Sattelfest: ".$_POST['sattelfest']."
	Sattelglück: ".$_POST['sattelglueck']."
	Schaler Zappenduster: ".$_POST['schalerzappenduster']."
	TECKiss: ".$_POST['teckiss']."
	TECKO: ".$_POST['tecko']."\n\n
	
	Folgende Individualisierungen werden gewünscht:\n
	Überschrift: ".$_POST['title']." ".$_POST['otherTitle']."\n
	mit Spruch: ".$_POST['slogan']." ".$_POST['otherSlogan']."\n
	mit Datum: ".$_POST['dateWedding']."\n\n
	Gewünschter Liefertermin: ".$_POST['dateDelivery']."\n\n
	
	Anmerkung: ".$_POST['message']."\n";

	$header = 'Content-type: text/html; charset=UTF-8' . "\r\n";
	
	mail($empfaenger, $betreff, $text, "From: $absendername <$absendermail>", $header);

	echo ('Vielen Dank! Wir melden uns schnellmöglich bei Ihnen!');
?>
